### DISCUSSION TITLE
Network Detector - Does it make any sense?

### DISCUSSION THREAD

**Discussion by rgriffogoes** [blakeblackshear/frigate#3336] [Ideas]
The scenario I was thinking is to have "main frigate" application running on a more powerful/existing server (e.g.: NAS) but consume a service for object detection running on a TPU enabled device (Coral [mini] Dev board).

That way the extra storage/cpu (and application stack compatibility) of the main server is used, but with the extra speed of the TPU hardware on the "detector node". 

If this make any sense, could anyone point me in the direction of where to look in the code?

Or I'm better off trying to run Frigate on my coral dev mini (even recompilling kernel if needed) and mounting a NFS share folder for extra storage ?

thanks and congrats on this AWESOME application!

**Comment by blakeblackshear** [comment]
You will never achieve acceptable latency. If you want to do this, you are better off using something like DOODS.

**Reply by rgriffogoes** [reply]
Thanks for the quick reply. I was kind of expecting that, but thanks for pointing out DOODS - probably wont use it for the camera but might be useful for some other ideas/projects
